infrastructure-playbooks/dashboard.yml

```
@@ -67,7 +67,7 @@
             status: "Complete"
             end: "{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}"
 
-- hosts: "{{ grafana_server_group_name }}"
+- hosts: '{{ groups["mgrs"] | default(groups["mons"]) }}'
```

```suggestion
- hosts: "{{ groups[mgr_group_name] | default(groups[mon_group_name]) }}"
```
but I'm not sure this is what we really wanna do here. I think we should stick to have the whole stack deployed in 1 node which would be a member of `grafana_server_group_name` (that could be either an mgr, a mon or anything).

Why would we absolutely want to enforce this to be deployed either on mgr or mon only?

@dsavineau let's discuss this as soon as you get online today, please.

---

@dsavineau I think we can keep `grafana_server_group_name` since we use delegate_to to execute the commands from the (mon|mgr)[0]; it can help us to logically keep the dashboard roles separated. 
+1 for the `run_once: true` option.